src/test/java/edu/harvard/iq/dataverse/api/MoveIT.java

```diff
@@ -254,6 +254,12 @@ public void testMoveLinkedDataset() {
         linkDataset.then().assertThat()
                 .statusCode(OK.getStatusCode());
 
+        // A dataset cannot be linked to the same dataverse again.
```

Comment:
is this test in the move tests? I see what you mean then - it works, but I wonder if we won't lose track that it's being tested here.

Reply:
Tests moved to their own class in 6bccd4f.